packages/react/src/components/IconButton/index.tsx

```diff
@@ -77,6 +77,11 @@ interface IconButtonProps
    */
   autoAlign?: boolean;
 
+  /**
+   * Optionally specify an href for your Button to become an `<a>` element
```

Comment:
Should these JSDoc comments say 'IconButton' instead of 'Button'?

```suggestion
   * Optionally specify an href for the IconButton to become an `<a>` element
```